src/api/images.tsx

```diff
@@ -114,3 +114,79 @@ export const createImageAlias = (
       .catch(reject);
   });
 };
+
+export const createImageFromInstance = (
```

Comment:
I think we should combine these functions into a single function `createImage`. This means we should try make adjustment to the existing `createImageFromInstanceSnapshot` function as well. The construction of the payload body should be determined outside of the API layer. Also be careful and don't forget about the `?project=` query parameter.

Reply:
Are you saying that createImageAlias should be integrated into both createImagefromInstanceSnapshot and createImageFromInstance?

Reply:
no I meant both `createImageFromInstanceSnapshot` and `createImageFromInstance` should become `createImage` with a body input determined from outside of the function.

Reply:
I think the `createImageFromInstance` function shouldn't be in this PR, it is introduced in https://github.com/canonical/lxd-ui/pull/833
though I agree, we could have a single `createImage` function, which receives a body. Then we can reuse it for the tarball upload and for the creation of an image from an instance.

Reply:
Created an issue to address this: https://warthogs.atlassian.net/browse/WD-14347?atlOrigin=eyJpIjoiMjcyYmI5Mjk3MjI1NGYxNWI4NWRlZGEwODViYjhlNDUiLCJwIjoiaiJ9

Reply:
Thanks